### ISSUE TITLE
Add Dropout functions to PyTorch Frontend

### ISSUE THREAD

**Issue by djl11** [unifyai/ivy#23]
Add [Dropout functions](https://pytorch.org/docs/stable/nn.functional.html#dropout-functions) to PyTorch frontend:

- [ ] dropout
- [ ] alpha_dropout
- [ ] feature_alpha_dropout
- [ ] dropout2d
- [ ] dropout3d

**Comment by aps19** [comment]
dropout #2663

**Comment by ryparmar** [comment]
- [] #6111

**Comment by AnnaTz** [comment]
- [ ] #6111

**Comment by ali-aboelezz** [comment]
#10320

**Comment by AnnaTz** [comment]
- [ ] #10320

**Comment by Immortalsoulslayer** [comment]
alpha_dropout #10549

**Comment by ali-aboelezz** [comment]
-[ ] #12418

**Comment by Rishikesh-Reddy** [comment]
- [x] #2760

**Comment by Ali-12122** [comment]
https://github.com/unifyai/ivy/issues/14671

**Comment by Ali-12122** [comment]
dropout2d #15394

**Comment by Kamil2601** [comment]
- [ ] #17405

**Comment by shaimah** [comment]
#17636

**Comment by abdullahpalaz** [comment]
dropout3d #18138

**Comment by akkefa** [comment]
alpha_dropout #18331

**Comment by fgokmenoglu** [comment]
feature_alpha_dropout [#19358](https://github.com/unifyai/ivy/issues/19358)

**Comment by FraBomba6** [comment]
- [ ] #21633

**Comment by FraBomba6** [comment]
- [ ] #21634
